packages/devextreme/js/__internal/utils/version.ts

```diff
@@ -35,16 +30,11 @@ export function parseVersion(version: string): Version {
 }
 
 export function assertDevExtremeVersion(packageName: string, version: string): void {
-  assertedVersions.push({
-    packageName,
-    version,
-  });
+  config({ assertedVersions: [...config()?.assertedVersions ?? [], { packageName, version }] });
 }
 
 export function clearAssertedVersions(): void {
-  /// #DEBUG
-  assertedVersions.splice(0);
-  /// #ENDDEBUG
+  config({ assertedVersions: [] });
```

Comment:
I'm not sure what these `/// #DEBUG` directives are here for, but I guess Ilya Kharchenko added them (I could not ever), so I guess they are here for a reason. Let's keep them.

Most probably, he marked methods that should only work during testing or something like that